### ISSUE TITLE
Slow running of P070+29

### ISSUE THREAD

**Issue by vmahat** [mhardcastle/ddf-pipeline#344]
Hello, 

I've been running the ddf-pipeline with standard LoTSS settings on P070+29.

It seems to hang, however, in one of the DI steps during deconvolution (image_ampphase1_di). The longest I've left this running is about 4 days during which nothing is written on disk -- I could wait a bit longer but this is probably an issue. 

I've been running the pipeline on the Herts cluster on one of the large nodes.

I've attached the latest log file and my config file for reference -- I noticed that the nearby P067+29 has been processed before. The only difference between the cfg file used for that pointing and P070+29 is the addition of NVSS for flux bootstrapping. There was a warning of diverging flux at the end of some of the major cycles, so I may re-run with NVSS information included as for P067+29. Note both of these fields contain the extremely bright 3C 123. Update on this will be posted, but any tips on the deconvolution hanging without progressing would be appreciated.

[image_ampphase1_di.log](https://github.com/mhardcastle/ddf-pipeline/files/13991017/image_ampphase1_di.log)
[tier1-jul2018_vijay.log](https://github.com/mhardcastle/ddf-pipeline/files/13991077/tier1-jul2018_vijay.log)

**Comment by vmahat** [comment]
Here are the corresponding bootstrap images with my P070+29 run without NVSS (left) and that from the P067+29 with NVSS in /beegfs/car/mjh/P067+29/ (right), centred on 3C 123.
<img width="960" alt="image" src="https://github.com/mhardcastle/ddf-pipeline/assets/77785349/45603464-1e1a-4255-a122-a9da418b5602">

**Comment by mhardcastle** [comment]
Hi Vijay, well I think it's clear why the deconvolution doesn't play nicely... 3C123's calibration is sufficiently screwed up that the mask is most likely filled with garbage.

The question is whether there's anything you can do to rescue this. Does the initial image (image_dirin_SSD_m.app.restored.fits) look comparable in quality to the one from the adjacent field?

**Comment by vmahat** [comment]
It looks like this 
<img width="960" alt="image" src="https://github.com/mhardcastle/ddf-pipeline/assets/77785349/7bb8832d-a7dc-4deb-81a3-46e4d1c5cd14">

**Comment by mhardcastle** [comment]
Yes, looks like the error is there in initial calibration. Could be that the sky model used for 3C123 in init cal is just not good enough...

**Comment by vmahat** [comment]
Is this an external model or that from image_dirin_SSD.app.model.fits ?

**Comment by mhardcastle** [comment]
I meant the sky model used in initial calibration with LINC (TGSS model probably). We redo the DI calibration but there's only so much you can fix...

**Comment by vmahat** [comment]
There are a few things I want to try so will keep this open until they're resolved

**Comment by vmahat** [comment]
Ah yes, this came about due to some improvements in the long baseline imaging with that fix. I'll have a go with that when it's merged

**Comment by vmahat** [comment]
Small update on this: hearing some rumours on the problems of demixing TauA, I processed the data through LINC without demixing TauA, and the resulting image_dirin_SSD_m.app.restored.fits looks appropriate (left) vs with demixing (right).
<img width="461" alt="image" src="https://github.com/mhardcastle/ddf-pipeline/assets/77785349/c2ddd8da-6405-49a9-8076-c1b9cd73c24c">
There are still artefacts/ripples across the field, that are probably due to the presence of TauA so this isn't the optimal solution -- given I'm interested in the long baselines I think this is okay for now.

**Comment by cyriltasse** [comment]
@vmahat There a simple option now in kMS to subtract all ATeam(+Sun) sources in one go, would you be willing to give it a try and compare?

**Comment by vmahat** [comment]
Sure, how does one specify this in the config?
